hera_cal/tests/test_firstcal.py

```diff
@@ -195,6 +195,14 @@ def test_UVData_to_dict(self):
                 np.testing.assert_equal(f[i, j][pol], np.resize(
                     uvd.flag_array[uvmask][:, 0, :, uvpol], f[i, j][pol].shape))
 
+    def test_UVData_to_dict_keys(self):
+        uvd = UVData()
+        uvd.read_miriad(os.path.join(DATA_PATH, 'zen.2457698.40355.xx.HH.uvcAA'))
+        nt.assert_equal(len(firstcal.UVData_to_dict([uvd])[0]), uvd.Nbls)
+        # reorder baseline array
+        uvd.baseline_array = uvd.baseline_array[np.argsort(uvd.baseline_array)]
```

Comment:
Check for equality instead of assigning to uvd.baseline_array?

Reply:
sorry, what do you mean by that? the baseline_array is purposely being reassigned b/c that is what causes the bug: when a baseline array is sorted in ascending order.

Reply:
This was basically getting at a test for my previous comment. But I think since the error comes about because the baseline array is sorted in this way, I'm ok with this test.